Toolchain.mk

```diff
@@ -0,0 +1,174 @@
+BIN = $(CURDIR)/bin
+$(BIN):
+	mkdir -p $(BIN)
```

Comment:
From my testing, the `$(BIN):` target is always executed regardless of the folder existing. /1

Reply:
From what I understand, that is now fixed.